animalfacts.py

```diff
@@ -1016,6 +1017,7 @@ def animalfactsbot(reddit):
     )
 
 DOLPHIN_FACTS = (
+	'Known for their Playful behavior, dolphins are highly Intelligent. They are as Smart as Apes, and the evolution of their larger brains is surprisingly similar to humans.',
```

Comment:
Can you fix the capital I please on intelligent? And the S on smart?

Reply:
Corrected!